# Message with information about the snapCircuits board state

Header header

# Information from each part in the circuit
# 

snap_circuits_part[] board
